release-notes.hbs.md

```diff
@@ -36,6 +36,15 @@ This release includes the following changes, listed by component and area.
 #### <a id='1-9-0-COMPONENT-NAME'></a> v1.9.0 Features: COMPONENT-NAME
  
 - Feature description.
+
+#### <a id='1-9-0-tanzu-developer-portal'></a> v1.9.0 Features: Tanzu Developer Portal
+
+- The Dora plug-in now has:
+  - New Date Range drop-down filters, Last 7 Days (default) and Last 30 Days
```

Comment:
We should call out the new default data range filter more explicitly.

Reply:
Updated